x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/metric_query.ts

```diff
@@ -39,7 +39,10 @@ export const getElasticsearchMetricQuery = (
   const to = timeframe.end;
   const from = timeframe.start;
   const offset = calculateDateHistogramOffset({ from, to, interval, field: timefield });
-  const offsetInMS = parseInt(offset, 10);
+  const offsetInMS =
```

Comment:
Can/should we do the aggType check before we even calculate the `offset` value so we can skip that calculation altogehter and just make `offset = 0` for those cases (and then the parseInt here will "just work")?